keps/sig-node/1797-configure-fqdn-as-hostname-for-pods/README.md

```diff
@@ -686,7 +692,7 @@ previous answers based on experience in the field._
 
 * **Will enabling / using this feature result in increasing size or count
   of the existing API objects?**
-  No
+Pods using this feature are required to set a new field, which increases the size of their objects.
```

Comment:
nit: Maybe add that this is just a couple bytes though.

Reply:
ok